### ISSUE TITLE
Sixel does not appear to work in WSL

### ISSUE THREAD

**Issue by Neo-Desktop** [contour-terminal/contour#1064]
### Contour Terminal version

0.3.11.258

### Installer source

GitHub: release page

### Operating System

Microsoft Windows [Version 10.0.22621.1265]

### Architecture

x86-64

### Other Software

**WSL2 - Ubuntu Jammy**

```
>wsl --version
WSL version: 1.1.3.0
Kernel version: 5.15.90.1
WSLg version: 1.0.49
MSRDC version: 1.2.3770
Direct3D version: 1.608.2-61064218
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22621.1265
```

```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.2 LTS
Release:        22.04
Codename:       jammy
```

**wslbridge2**

```
>wslbridge2.exe --help

wslbridge2 v0.10 : Runs a program within a Windows Subsystem for Linux (WSL) pty.
Copyright (C) 2019-2022 Biswapriyo Nath.
Licensed under GNU General Public License version 3 or later.
```

### Steps to reproduce

cloned https://github.com/csdvrx/sixel-testsuite
ran `cat snake.six`

### Expected Behavior

A picture of a snake rendered with sixel as seen when running xterm in WSL or wsltty

### Actual Behavior

no output

### Additional notes

My `contour.yml` is the default config aside from the following:

```yaml
profiles:
    main:
        shell: "C:\\wsl-bridge2\\wslbridge2.exe"
        arguments: ["-d", "ubuntu", "-W", "~", "-e", "TERM", "-e", "COLORTERM"]
        environment:
            TERM: contour
            COLORTERM: truecolor
```

using the default `wsl` executable doesn't appear to work due to it effectively being funneled through windows's `conhost` (or so I understand)

[wsltty](https://github.com/mintty/wsltty) seems to handle things correctly using [wslbridge2](https://github.com/Biswa96/wslbridge2) I'm basically trying to replicate that in contour and am unable to do so

it's also possible this is user error but as far as I can tell it's a near-identical situation (between wsltty and contour), the issue being that contour doesn't seem to be processing the sixel escape codes

any light that can be shed on this would be very helpful

Thanks!

**Comment by christianparpart** [comment]
Hey; oh interesting. I would need to test this, but as far as ConPTY is concerned (which is used under the hood), We (Contour, and TEs in general that are using ConPTY under the hood) are hard limited to the features ConPTY supports. This may change in the future, depending on the progress on the other side (https://github.com/microsoft/terminal/issues/1173).

I would have to give it a try what `wslbridge2` is doing here, and I'd actually **love** to find out how to bypass ConPTY and still make it work with WSL as well as a locally running CMD or PowerShell. Putty is not suffering from this because it's purely networked, and mintty is based on the POSIX layer emulation (Cygwin). I hope we can avoid Cygwin and still unblock all the features non-Windows platform Contour's have. 🤔

**Comment by christianparpart** [comment]
This can **soon** be worked around with the builtin SSH client support. So you can directly SSH into your WSL from within Windows GUI. (#1323)

**Comment by christianparpart** [comment]
I think we can close this ticket now, because we have builtin SSH support merged (#1323) which helps displaying Sixel images in Contour on Windows.

NB: I was just yesterday trying WSLg and successfully ran Contour in a Ubuntu, displaying it on Windows 10 natively. Maybe you can give this a try as well? I'd be happy if you'd report it back here, if that one was successful for you.

The limitation due to ConPTY / Conhost not supporting Sixels just yet. We've to see when https://github.com/microsoft/terminal/issues/1173 will be merged.
